pkg/sbom/sbom_test.go

```diff
@@ -0,0 +1,155 @@
+//go:build integration
+// +build integration
+
+package sbom
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"chainguard.dev/melange/pkg/cli"
+)
+
+var (
+	updateGoldenFiles = flag.Bool("update-golden-files", false, "update golden files")
+)
+
+func TestGenerate(t *testing.T) {
+	const wolfiAPKRepoURL = "https://packages.wolfi.dev/os"
+
+	testArtifacts := []string{
+		"crane-0.19.1-r6.apk",
+
+		// Note: Syft is producing nondeterministic results for these APKs, so the tests
+		// aren't stable. Uncomment them when the issue is resolved.
+		//
+		// "jenkins-2.461-r0.apk",
```

Comment:
Is there an issue tracking this that we could include here so we know how to keep track of this work?

Reply:
Great callout — I'll open an issue and add the reference to the comment here when I open this PR's companion for vuln scan integration tests.